metricbeat/module/apache/status/data.go

```diff
@@ -16,41 +17,41 @@ var (
 	// This should match: "CPUSystem: .01"
 	matchNumber = regexp.MustCompile("(^[0-9a-zA-Z ]+):\\s+(\\d*\\.?\\d+)")
 
-	schema = h.NewSchema(common.MapStr{
-		"total_accesses":    h.Int("Total Accesses"),
-		"total_kbytes":      h.Int("Total kBytes"),
-		"requests_per_sec":  h.Float("ReqPerSec", h.Optional),
-		"bytes_per_sec":     h.Float("BytesPerSec", h.Optional),
-		"bytes_per_request": h.Float("BytesPerReq", h.Optional),
-		"workers": common.MapStr{
-			"busy": h.Int("BusyWorkers"),
-			"idle": h.Int("IdleWorkers"),
+	schema_ = schema.Schema{
```

Comment:
All in one package wouldn't be possible without significant repetition between `mapstrstr` and `mapstriface`. What do you mean by re-export. I tried something like `type Schema schema.Schema` but that doesn't really work because the resulting type is not compatible.